sbin/pfdhcplistener

```diff
@@ -198,13 +190,26 @@ sub process_pkt {
         my $l2 = NetPacket::Ethernet->decode($pkt);
         my $l3 = $l2->{type} eq ETH_TYPE_IP ? NetPacket::IP->decode($l2->{'data'}) : NetPacket::IPv6->decode($l2->{'data'});
         my $l4 = NetPacket::UDP->decode($l3->{'data'});
+        my $udp_payload_b64 = MIME::Base64::encode($l4->{data});
+        my %args = (
+            src_mac => clean_mac($l2->{'src_mac'}),
+            dest_mac => clean_mac($l2->{'dest_mac'}),
+            src_ip => $l3->{'src_ip'},
+            dest_ip => $l3->{'dest_ip'},
+            running_w_dhcpd => $running_w_dhcpd,
+            is_inline_vlan => $is_inline_vlan,
+            interface => $interface,
+            interface_ip => $interface_ip,
+            interface_vlan => $interface_vlan,
+            net_type => $net_type,
+        );
+        my $apiclient = pf::api::jsonrpcclient->new;
```

Comment:
The only reason is if we want to have more "complicated" logic for creating the api client.

pf::api::queue->new({queue => "pfdhcplistener"});#To override the default queue general